docs/guide/write/quick-start.md

```diff
@@ -1,9 +1,104 @@
-# Coming Soon
+# Writing Tests Quick Start 
 
-::: info
-This page is Under Construction 🏗️, whilst Moonwall has been built - the docs have not.
+Let's write a super simple test case that we'll run in the 
+[Quick Start Running Tests](/guide/test/quick-start) Section. 
 
-Please Stay tuned for more details of this feature!
-:::
+### Moonwall Utils
 
-![underconstruction](/under-construction.png)
+Moonwall utils is a utils package with helpful constants and various functions for Moonwall. In the test file you can refer to some pre-funded development accounts like ALITH and BALTATHAR which can be easily imported from ```@moonwall/util```. [Moonwall Utils](https://github.com/Moonsong-Labs/moonwall/tree/main/packages/util){target=blank} contains tons of helpful constants, classes, functions, and helpers. For example, [`Chain.ts`](https://github.com/Moonsong-Labs/moonwall/blob/main/packages/util/src/constants/chain.ts){target=blank} includes weights, precompile addresses, gas constants, and more. 
```

Comment:
Maybe make reference that ALITH and BALTHATHAR are EVM versions of alice & bob.
- Account in substrate is 32 bytes
- Account20 (i.e. evm addresses) are 20 bytes

This is why we use non-standard names of ALITH not ALICE, to differentiate between the two schemes.